pkg/internal/session/session.go

```diff
@@ -216,7 +218,7 @@ func notDeleted(condition *istiov1alpha1.Condition) bool {
 }
 
 func deploymentOrDeploymentConfig(condition *istiov1alpha1.Condition) bool {
-	return condition.Source.Kind == "DeploymentConfig" || condition.Source.Kind == "Deployment"
+	return condition.Source.Kind == k8s.DeploymentKind || condition.Source.Kind == openshift.DeploymentConfigKind
```

Comment:
Originally done to avoid a package import on client of the server side.. but.. it's the same binary at this point anyway

Reply:
Ok, didn't think about that.